### DISCUSSION TITLE
Should we move to use jsx component files?

### DISCUSSION THREAD

**Discussion by PatrickMatthiesen** [PhilipFlyvholm/learnit-plus-plus#22] [Ideas]
We have been wondering if it was worth it to use .jsx to simplify and clean up the code a bit.

Does anyone have any thoughts on this?

**Comment by PhilipFlyvholm** [comment]
This should be possible with the vite plugin but I can't seem to find how to inject the .jsx into a page. We would probably need to inject react into the webpage.

**Reply by PhilipFlyvholm** [reply]
I found this example the uses the current vite plugin to maybe solve it, however it seems hacky.

[https://github.com/yosevu/react-content-script](https://github.com/yosevu/react-content-script)
